crates/bevy_audio/src/audio.rs

```diff
@@ -300,38 +53,73 @@ impl VolumeLevel {
     }
 }
 
-/// Settings to control playback from the start.
-#[derive(Clone, Copy, Debug)]
+/// How should Bevy manage the sound playback?
+#[derive(Debug, Clone, Copy)]
+pub enum PlaybackMode {
+    /// Play the sound once. Do nothing when it ends.
+    Once,
```

Comment:
Yes you could absolutely do that. The typical intended use of `Once` is for entities that do more than just play a sound. Remember, now that sounds are entities (or components rather), you can have any combination of components on your entities. You don't necessarily need to have a separate entity just for the sound.

You could, for example, have a "door" object, which just exists in the world, carries a `PbrBundle` to render some graphics, and you just insert an `AudioBundle` onto it when you want to play sound. You might prefer that over spawning a separate entity for the sound, because you get to reuse the same Entity id, which might make your code/logic/queries simpler.

If you are spawning a dedicated entity for the sound, then you should probably just use `Despawn` instead of `Once`.

Reply:
Okay, I understand the implications now. Thanks.